AirCasting/SessionsSynchronization/Controller/SessionSynchronizationController.swift

```diff
@@ -34,18 +31,18 @@ final class SessionSynchronizationController: SessionSynchronizer {
         self.store = store
     }
     
-    func triggerSynchronization(completion: (() -> Void)?) {
+    func triggerSynchronization(logout: Bool = false, completion: (() -> Void)?) {
```

Comment:
plase don't use bools for such scenarios. Also please don't spill domain naming of other parts of the app to classes that don't really need to know about it. Why would sync controller know anything about logging out? `skipDownloads: Bool = false` would be a much more contained IMO.